src/components/calendar/MeetingModal.js

```diff
@@ -43,7 +43,10 @@ const MeetingModal = props => {
 
   const submitMeeting = e => {
     e.preventDefault();
-    let newParticipants = participants.map(participant => participant.uid);
+    let newParticipants = [];
+    if (participants.length) {
```

Comment:
Added condition so the map doesn't fail if no additional participants are added.